misc/sample_application/image_builder/create_image.sh

```diff
@@ -91,6 +91,11 @@ function copy_and_enlarge_image() {
         growpart ${dest_image} 1
 }
 
+function dump_proxy() {
+	local temp_file=$(mktemp /tmp/proxy.XXXX)
+	sudo apt-config dump | grep -i proxy > ${temp_file} 2>&1
+	sudo mv ${temp_file} proxy.conf
+}
 function resizing_guest_root() {
```

Comment:
One empty line between adjacent function blocks, please.

Reply:
Add the empty line in the function thanks.